esp/esp/program/modules/base.py

```diff
@@ -389,9 +365,6 @@ def isCompleted(self):
     def prepare(self, context):
         return context
 
-    def getNavBars(self):
```

Comment:
This appears to have been a way modules could do something magical with navbars.  In any case, despite a number of modules implementing the method, it doesn't appear to be used.